medium-tests/src/test/java/mediumtest/hotspots/HotspotLocalDetectionSupportMediumTests.java

```diff
@@ -84,7 +84,7 @@ void it_should_not_support_local_detection_in_standalone_mode() {
 
     assertThat(checkResponse)
       .extracting(CheckLocalDetectionSupportedResponse::isSupported, CheckLocalDetectionSupportedResponse::getReason)
-      .containsExactly(false, "The project is not bound, please bind it to SonarQube 9.9+ or SonarCloud");
+      .containsExactly(false, "The project is not bound, please bind it to SonarQube (Server, Cloud)");
```

Comment:
I've read the branding guide but reading this feels super bad.

Reply:
I agree, maybe here we could afford to spell it out `SonarQube Server or SonarQube Cloud`. But not a blocker from my perspective.